srcs/server/src/game/entities/game.entity.ts

```diff
@@ -0,0 +1,33 @@
+import { GameGrid } from './gameGrid.entity';
+import { Player } from './player.entity';
+import { Client } from './client.entity';
+import { GamePhysics } from './gamePhysics.entity';
+
+export class Game {
+  constructor(id: string) {
+    this.gameGrid = new GameGrid();
+    this.gamePhysics = new GamePhysics();
+    this.players = [];
+    this.viewers = [];
+    this.id = id;
+    this.status = 0;
+  }
+
+  id: string;
+  status: 0 | 1 | 2 | 3 | 4;
```

Comment:
enum ?

Reply:
It is strange but I have errors with typescript. If this is an enum, then I will be forced to type game as "any" because it will not match my enum type with the values I will assign to status, even if this is in range.

Maybe I should spend more time trying to solve this to have a better typing globally